registry/server/common/middlewares/validateRequest.ts

```diff
@@ -17,19 +19,16 @@ interface ValidationConfig {
     selector: SelectDataToValidate,
 }
 
-const validateRequestFactory = (validationConfig: ValidationConfig[]) => async (
-    req: Request,
-    res: Response,
-    next: any,
-) => {
+const validateRequestFactory = (validationConfig: ValidationConfig[]) => async (req: Request, res: Response, next: NextFunction) => {
     try {
         await Promise.all(_.map(
             async ({schema, selector}) => schema.validateAsync(selector(req), {abortEarly: false}),
             validationConfig
         ));
         next();
-    } catch (e) {
-        res.status(422).send(preProcessErrorResponse(e));
+    } catch (error) {
+        console.error(`Caught an error while validating request data for '${req.url}':`, error);
+        res.status(422).send(preProcessErrorResponse(new Error(preProcessError(error))));
```

Comment:
This line is super hard to understand....

Reply:
Removed